
<launch>


	<node pkg="main_loop" type="path_planning_" name="path_planning_" output="screen" respawn="true">
	</node>

	<node pkg="main_loop" type="main_with_class" name="main_with_class" output="screen" respawn="true">
	</node>
	<node pkg="main_loop" type="agent" name="agent" output="screen">
	</node>
	
	
	
	   <include file="$(find lidar_2020)/launch/octagonal_range.launch">
    </include>	
	
    


</launch>
